### ISSUE TITLE
Doesn't work with Astro 4

### ISSUE THREAD

**Issue by mashehu** [astro-community/md#10]
Similar fix to #8 needed, I guess
```
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: nf-co.re@0.0.1
npm WARN Found: @astrojs/markdown-remark@4.0.0
npm WARN node_modules/@astrojs/markdown-remark
npm WARN   @astrojs/markdown-remark@"4.0.0" from @astrojs/mdx@2.0.0
npm WARN   node_modules/@astrojs/mdx
npm WARN     @astrojs/mdx@"2.0.0" from the root project
npm WARN   2 more (astro, the root project)
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer @astrojs/markdown-remark@"^3" from @astropub/md@0.3.0
npm WARN node_modules/@astropub/md
npm WARN   @astropub/md@"^0.3.0" from the root project
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: nf-co.re@0.0.1
npm WARN Found: astro@4.0.0
```

**Comment by tiwka19** [comment]
@jonathantneal please accept the PR

**Comment by jonathantneal** [comment]
@tiwka19, the PR has been accepted and the issue has been fixed in release 0.4.0.

Thank you, everyone, for your help and patience. 🙏
